cpp/perspective/src/include/perspective/sym_table.h

```diff
@@ -31,8 +31,8 @@ class t_symtable {
     t_mapping m_mapping;
 };
 
-const char* get_interned_cstr(const char* s);
-t_tscalar get_interned_tscalar(const char* s);
-t_tscalar get_interned_tscalar(const t_tscalar& s);
+PERSPECTIVE_EXPORT const char* get_interned_cstr(const char* s);
```

Comment:
@sc1f this is currently being exposed as part of `libpsp`'s public interface (it is used in the python cpp code). should this be public?